hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/hudi/HoodieWriterUtils.scala

```diff
@@ -92,10 +92,9 @@ object HoodieWriterUtils {
    * @return
    */
   def getPartitionColumns(parameters: Map[String, String]): String = {
-    val props = new TypedProperties()
+    val props = new Properties()
```

Comment:
why need this change, TypedProperties should act as same as Properties

Reply:
Because HoodieSparkUtils.getPartitionColumns use `Properties` the input parameter type.